groovy/postInit/mod/GregTechFoodOption.groovy

```diff
@@ -419,7 +419,7 @@ LCR.recipeBuilder()
     .buildAndRegister()
 
 BR.recipeBuilder()
-    .notConsumable(metaitem('dustSodiumHydroxide'))
+    .notConsumable(metaitem('dustSodiumHydroxide') * 6)
```

Comment:
I would change this to 2000 sodium hydroxide solution as this reaction is carried out in aqueous media.